datumaro/plugins/data_formats/cifar.py

```diff
@@ -203,7 +203,8 @@ def _find(path: str, meta_file_name: str) -> List[Dict[str, Any]]:
                     # Subset files have no extension in the format, and
                     # should not be the meta file.
                     file_filter=lambda p: not osp.splitext(osp.basename(p))[1]
-                    and osp.basename(p) != meta_file_name,
+                    and osp.basename(p) != meta_file_name
+                    and osp.basename(p) != "hash_key_meta",
```

Comment:
Is it necessary? Is "hash_key_meta" directory, not file?

Reply:
Yes, `hash_key_meta` is directory. In cifar, it consists of like `batches.meta` and `test`. I added it because I thought it would be confusing if there is `hash_key_meta` because only `test` should be recognized. If this part is missing or replaced, an error will occur.

Reply:
I thought that it will only check files not directories because it is "`file_filter`", but this seems checking directories also. I guess that you need to add `not osp.isdir(p) and` in front of it rather than adding `and osp.basename(p) != "hash_key_meta"` to the last.

Reply:
Fix in [4113433](https://github.com/openvinotoolkit/datumaro/commit/411343324cd5f318a511b27f9969e2f69b6dadc0)